README.md

```diff
@@ -351,3 +351,33 @@ example:
 
     you can download the full output from:
     https://<ansible tower instance>/api/v1/jobs/20895/stdout/?format=txt_download
+
```

Comment:
add a link to this section: "Tower companion provides the following command line scripts:"...